### DISCUSSION TITLE
可以給播放列表的右鍵菜單加入下一首播放嗎

### DISCUSSION THREAD

**Discussion by StealthEmissarI** [zhongyang219/MusicPlayer2#453] [Ideas]
![image](https://user-images.githubusercontent.com/61230741/198073897-01381d8e-0f57-4672-b2a2-d354c9b6b306.png)

**Comment by zhongyang219** [comment]
下一首播放的功能已经添加了，请等待下个版本更新。或者在[Actions](https://github.com/zhongyang219/MusicPlayer2/actions)页面下载最新的预构建版本。
